cmake/Modules/CocosBuildHelpers.cmake

```diff
@@ -66,28 +66,22 @@ function(cocos_mark_resources)
 endfunction()
 
 # mark the files in the sub dir of CMAKE_CURRENT_DIR
-function(cocos_mark_source_files cocos_target)
+function(cocos_mark_code_files cocos_target)
```

Comment:
Is this code is for creating groups in xcode and Visual Studio  like i.e.: https://stackoverflow.com/a/33813154/4514853  ?
If yes I think that we should remove it.  Why we needs this groups at all? Both editor can display files using current folders structure. It will make only cmake file complicated and harder to maintain. If we think that folders structure is not as good as it should be we should change folder structure instead of creating filters/groups for xcode/visual studio editor